lib/sinatra/base.rb

```diff
@@ -1849,7 +1848,6 @@ class << self
       server.unshift 'reel'
       server.unshift 'puma'
       server.unshift 'mongrel'  if ruby_engine.nil?
-      server.unshift 'thin'     if ruby_engine != 'jruby'
       server.unshift 'trinidad' if ruby_engine == 'jruby'
```

Comment:
Just in case I would like to ask you, do we need to add rainbows here? AFAIK it's not necessary cuz rainbows does not provide the rack handler by default. Is this correct?

Reply:
I agree with you about not adding rainbows.